### PR TITLE
Overlay

### PR DESCRIPTION
**Pull request by andperks6**
This includes functionality for relative indoor floor selection when users click the 'indoor' button.

### CODE REVIEW

File: public/styles/relativeFloorSelect.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.RFSContainer {
```
Comment (blakemorgan) [inline public/styles/relativeFloorSelect.css]: What is this for?

Comment (andperks6) [reply public/styles/relativeFloorSelect.css]: The container for the relative floor select, there was a weird box shadow around the widget that I was trying to fix but ended up not needing it

Comment (blakemorgan) [reply public/styles/relativeFloorSelect.css]: Are you able to remove it? Or does it need to be there empty still?

Comment (andperks6) [reply public/styles/relativeFloorSelect.css]: yeah removed it, just had to figure out how to edit a PR file

**Pull request merged**